server/flaskr/services/notifications/handlers/meeting_started_notification_handler.py

```
@@ -0,0 +1,57 @@
+
+    def remove(self, _: list[NotificationJob]):
+        assert False, "Can't remove instant notifications"
+
+    def get_notifications_from_job(self, job: NotificationJob):
+        set_tenant(job.orgId)
+        usersIds = MeetingDataHandler.get_meeting_users(job.targetId)
+
+        notifications = []
+        for userId in usersIds:
+            token, locale = UserDataHandler.get_user_fcm(userId)
+            if token is None: continue
```

Ruff (0.15.21)

[error] Multiple statements on one line (colon)

(E701)